CONTRIBUTING.md

```diff
@@ -373,6 +374,150 @@ regression    =   1.44 sec*proc (1 test)
 Total Test time (real) =   1.47 sec
 ```
 
+#### Adding Performance Regression Tests
+
+Regression tests can also be added for the purposes of testing the performance.
+Adding performance tests does require understanding a bit about how the ci
+infrastructure is setup. Currently, the performance tests are setup to run on
+dedicated Power9 nodes that are provided by LANL, this is to ensure consistency
+in the result. However, because the performance tests are running on LANL
+machines it means that it is not straightforward to make the results viewable
+to non LANL contributors. To make the results of the regression tests viewable
+on github a custom github application was developed to be used with Parthenon
+which we will call the Parthenon Performance Metrics App (PPMA).
+
+The PPMA is series of python scripts located in the parthenon repository in
+./scripts/python/packages/parthenon_performance_app. The PPMA has the following
+responsibilities:
+
+1. Submit a status check associated with the commit being analyzed to the
+   Parthenon repository
+2. Read in performance metrics from regression tests in a build directory
+3. Compare the current metrics with metrics stored in the parthenon.wiki, the
+   files in the wiki should contain performance metrics from previous runs.
+4. Generate images showing the performance numbers.
+5. Add images to an orphan branch in the parthenon repository (branch name is
+   figures).
+6. Add new performance metrics to the appropriate performance .json file stored
+   in the wiki. These json files are labeled with the following format:
```

Comment:
```suggestion
   in the Wiki. These JSON files are labeled with the following format:
```
Would prefer capitalized JSON everywhere (also instead of ".json" except when referring to a specific file)